packages/api/amplify_api/lib/src/graphql/web_socket/blocs/web_socket_bloc.dart

```diff
@@ -175,8 +206,8 @@ class WebSocketBloc with AWSDebuggable, AmplifyLoggerMixin {
     }
   }
 
-  // Receives connection ack message from ws channel.
-  // Moves ConnectingState => ConnectedState, and registers sub blocs
+  /// Receives connection ack message from ws channel.
+  /// Moves ConnectingState => ConnectedState, and registers sub blocs
```

Comment:
Through some wifi on/off toggling in web and unsub/sub I was able to consistently trigger an assertion error from line 214 where the current state is connected. Not sure if this is a blocker bc requires pretty specific order of operations, just a callout.